### PR TITLE
Fix wireframe render for meshes with special properties

### PR DESCRIPTION
**Pull request by Atlante45**
Turning off all parameters of the render key for wireframes.
That reduces the number of pass for wireframes from 16 to 1.
@samcake can you confirm this makes sense for all of them?

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 52397bf87af1ca5d2431f108cf55dfadb207a5fe available at: https://gist.github.com/5886df450384e4f232f3<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4801.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4801.dmg

File: libraries/render-utils/src/Model.cpp [hunk 1]
```diff
@@ -2068,6 +2068,12 @@ void Model::segregateMeshGroups() {
         bool hasSpecular = mesh.hasSpecularTexture();
         bool hasLightmap = mesh.hasEmissiveTexture();
         bool isSkinned = state.clusterMatrices.size() > 1;
+        bool isWireframe = this->isWireframe();
```
Comment (ZappoMan) [inline libraries/render-utils/src/Model.cpp]: why this-> ?

Comment (ZappoMan) [inline libraries/render-utils/src/Model.cpp]: use a different variable name and call the method directly...

**Comment by hifi-gustavo** [comment]
Build console output for commit: e51040db5302a2d752d71ed70581a733b250613c available at: https://gist.github.com/5886df450384e4f232f3<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4801.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4801.dmg

**Pull request merged**